boards/x86/intel_adl/up7000.dts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2023 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "intel_adl.dts"
+
+/ {
+	model = "UP 7000 board";
+	compatible = "intel,up7000";
```

Comment:
is the vendor "UP" or "Intel"? You have something different here than in the up7000.yaml. Or should it be AAEON?

Reply:
Probably vendor is `up`, I will change this way.

Reply:
> Probably vendor is `up`, I will change this way.

kk. Would need to add it in vendor-prefixes too, then.

Reply:
> is the vendor "UP" or "Intel"? You have something different here than in the up7000.yaml. Or should it be AAEON?

That's a good point. It's definitely not Intel (although a lot of the underlying hardware is). Looking at the up-board.org pages, it seems like "UP" (or more correctly "UP Bridge the Gap") is a brand from AAEON that groups together this kind of platforms. So probably the right answer for "who is the vendor?" is "AAEON"?

Reply:
Btw, here's the relevant page on AAEON's own website: https://www.aaeon.com/en/ai/up-bridge-the-gap

Reply:
> So probably the right answer for "who is the vendor?" is "AAEON"?

Yes that would be my guess too, but you guys can probably help figure out the most definitive/appropriate answer to that question ;)

Reply:
As for the board name, `up7000` seems nice and short, and although it's in a board directory called `intel_adl` I think that's the right choice, at least for now, since it's so similar to Intel's own Alder Lake reference boards. Putting it anywhere else would mean a lot more duplication in the board definition files (Kconfig, DTS, etc).